tests/test_pipeline.py

```diff
@@ -7,7 +7,7 @@
 
 class Counter(Producer):
     def __init__(self):
-        super(Producer, self).__init__()
```

Comment:
Use of `super(Klass, self).__init__()` is hereby forbidden. (This is a bug.)

Reply:
oh